README.rst

```diff
@@ -18,7 +18,145 @@ SpringField is ideal for:
 - Parsing CSV data structures from ``csv.DictReader``
 - Turning anything Python can parse into a ``dict`` or ``list`` into a structured object
 
+There is also a helper library for using SpringField with Mongo: `springfield-mongo <https://github.com/six8/springfield-mongo>`_
+
+
+Quickstart
+----------
+
+To define an ``Entity``, subclass ``springfield.Entity``. Define your attributes by specifying
+``fields``. This library provides the follow self-describing fields to start with:
+
+- ``IntField``
+- ``FloatField``
+- ``BooleanField``
+- ``StringField``
+- ``BytesField``
```

Comment:
This doesn't actually exist until https://github.com/six8/springfield/pull/11/files is merged